make-util.js

```diff
@@ -18,7 +18,67 @@ var makeOptions = require('./make-options.json');
 // list of .NET culture names
 var cultureNames = [ 'cs', 'de', 'es', 'fr', 'it', 'ja', 'ko', 'pl', 'pt-BR', 'ru', 'tr', 'zh-Hans', 'zh-Hant' ];
 
-function assert(value, name) {
+//------------------------------------------------------------------------------
+// shell functions
+//------------------------------------------------------------------------------
```

Comment:
`set('-e')` isnt available in shelljs 0.3, so i added wrapper functions that shell.error() and throw